internal/model/update.go

```diff
@@ -52,8 +52,7 @@ type MarkAsProcessed struct {
 }
 
 type RecordEcosystemVersions struct {
-	Ecosystem       string         `json:"ecosystem" yaml:"ecosystem"`
-	PackageManagers map[string]any `json:"package-managers,omitempty" yaml:"package-managers,omitempty"`
+	EcosystemVersions map[string]any `json:"ecosystem_versions" yaml:"ecosystem_versions"`
```

Comment:
Do we still want the omitempty?

Reply:
I don't _think_ so. IIUC, this struct is only hit when the endpoint is called, in which case this field is expected to be present.